python/tvm/auto_scheduler/search_task.py

```diff
@@ -481,7 +481,7 @@ def __init__(
             desc,
         )
 
-    def tune(self, tuning_options, search_policy=None):
+    def tune(self, tuning_options, search_policy=None, adaptive_training=False):
```

Comment:
probably we dont need to adjust auto_scheduler's interface

Reply:
This only impacts `tune_te`, which does not call thru `TaskScheduler.tune` but `SearchTask.tune` instead. If we don't want to enable that for `tune_te` I can just remove that change.

Reply:
i see...that makes sense. let's keep the flag here for tune_te then :-)